docs/source/1.0/spec/aws/aws-core.rst

```diff
@@ -1311,3 +1332,4 @@ existing AWS services.
 .. _Amazon Resource Name (ARN): https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html
 .. _AWS Service Namespaces: https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#genref-aws-service-namespaces
 .. _CloudFormation resource type: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-template-resource-type-ref.html
+.. _AWS Regions and Endpoints:
```

Comment:
Missing the URL.  Should be https://docs.aws.amazon.com/general/latest/gr/rande.html.

Reply:
Whoops, thanks! I had left that empty because my internet was down at the time and it appears I just forgot to add it once the internet came back on.